tests/search/test_main.py

```diff
@@ -16,14 +16,34 @@ def test_index(
 
     # load page and establish section is visible
     page.goto("http://localhost:3000")
-    section = page.get_by_test_id("search-section")
+    section = page.get_by_test_id("search-results-section")
     expect(section).to_be_visible()
     page.screenshot(path="tests_search_test_main-test_index-on-load.jpeg")
 
     # check heading is showing
-    heading = page.get_by_test_id("search-heading")
+    heading = page.get_by_test_id("search-results-heading")
     expect(heading).to_be_visible()
-    assert heading.inner_text() == "Search"
+    assert heading.inner_text() == "showing 7 of total 7 items found"
+
+    # check sidebar is showing
+    sidebar = page.get_by_test_id("search-sidebar")
+    expect(sidebar).to_be_visible()
+
+    # check search input is showing
+    search_input = page.get_by_placeholder("Search here...")
+    expect(search_input).to_be_visible()
```

Comment:
maybe two more tests here, one where you search for something that is actually findable like "mex" and assert you get 1 result, then search for "blablabla" and check you get 0. each assertion with a screenshot afterwards, so we can better debug it in pipelines.

Reply:
updated in 5802b4a6b14bd7c864c8a8599e727917b77ec42d